### ISSUE TITLE
problem with access to minikeepass on iPhone

### ISSUE THREAD

**Issue by stefaniascagliola** [MacPass/MacPass#1177]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by stefaniascagliola** [comment]
Dear all, 

I have been using the minikeepass for ages on my iPhone, knowing that there was no way of retrieving the content if I would forget the password, the last months I would get the message that the App Store would no longer support the App, but it would still open and I could use it, I had no clue it would be a temporary status, today suddenly it does no longer appear I have tried to retrieve it through the purchase history of the App Store, but I didn't get it through there, I believe I downloaded it from an online service advised to me by a friend. Any idea of how to retrieve the content? All my pass words are in there!!! Does anyone know the problem and have a solution that I could try out? 
thanks for your ideas on this!
Stef from Rotterdam Holland,
